src/SourceBuild/content/eng/Versions.props

```diff
@@ -24,7 +24,7 @@
       or minor release, prebuilts may be needed. When the release is mature, prebuilts are not
       necessary, and this property is removed from the file.
     -->
-    <PrivateSourceBuiltArtifactsPackageVersion>0.1.0-8.0.100-5.centos.8-x64</PrivateSourceBuiltArtifactsPackageVersion>
+    <PrivateSourceBuiltArtifactsPackageVersion>8.0.100-preview.2.23157.25</PrivateSourceBuiltArtifactsPackageVersion>
```

Comment:
It looks like the artifacts were uploaded with a stable version versus the full version.

Reply:
Fixed, thanks.